_tips/writing.id.md

```diff
@@ -0,0 +1,470 @@
+  * [Guna Tautan (Hanya Tautan) 2.4.9](/WAI/WCAG21/quickref/#link-purpose-link-only) ([Memahami 2.4.9](/WAI/WCAG21/Understanding/link-purpose-link-only))
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Tulis alternatif teks yang bermakna untuk gambar
+
+Untuk setiap gambar, tulislah alternatif teks yang menjelaskan informasi atau fungsi dari gambar tersebut. Untuk gambar yang sifatnya hanya dekoratif, tidak perlu menuliskan teks alternatif.
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Contoh: Gunakan alternatif teks untuk mengkomunikasikan informasi yang penting" class="example" %}
+{:/}
+
+<div class="text-alt two-column">
+  <figure>
+    <figcaption>{% include_cached icon.html name="ex-circle" label="Salah" %} Tidak informatif</figcaption>
+    <div>
+      {% include_cached img.html src="phone_charging.png" alt="Mengisi daya ponsel"  float="left" %}
+      <div>
+        <p> Mengisi daya ponsel: Hubungkan ponsel ke stopkontak menggunakan kabel dan adaptor daya yang disediakan.</p>
+        <p><strong>Alternatif teks untuk gambar</strong>: "Mengisi daya ponsel"</p>
+      </div>
+    </div>
+  </figure>
+  <figure>
+    <figcaption>{% include_cached icon.html name="check-circle" label="Oke" %} Informatif</figcaption>
+    <div>
+      {% include_cached img.html src="phone_charging.png" alt="Colokkan kabel pada bagian bawah ponsel." float="left" %}
+      <div>
+        <p>Mengisi daya ponsel: Hubungkan ponsel ke stopkontak menggunakan kabel dan adaptor daya yang disediakan.</p>
+        <p><strong>Alternatif teks untuk gambar</strong>: "Colokkan kabel pada bagian bawah ponsel"</p>
+      </div>
+    </div>
+  </figure>
+</div>
+<p class="note">Alternatif teks biasanya tidak terlihat; teks disertakan pada contoh ini agar Anda dapat melihatnya.
+</p>
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Informasi Lebih Lanjut" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [Konten Nonteks 1.1.1](/WAI/WCAG21/quickref/#non-text-content) ([Memahami 1.1.1](/WAI/WCAG21/Understanding/non-text-content))
+* **Tutorial**
+  * [Gambar](/tutorials/images/)
+* **User Story**
+  * [Menjelaskan nilai dari alternatif teks bagi pengguna penyandang disabilitas netra](/people-use-web/user-stories/#accountant)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Buat transkripsi dan takarir untuk multimedia
+
+Untuk konten audio, seperti podcast, berikan transkripsinya. Untuk konten audiovisual, seperti video pelatihan, sediakan pula takarirnya. Cantumkan pada transkripsi dan takarir informasi terkait ucapan lisan dan suara yang penting diketahui untuk memahami konten, misalnya, 'pintu berderit'. Untuk transkripsi video, sertakan juga deskripsi konten visual yang penting, misalnya 'Athan meninggalkan ruangan'.
```

Comment:
Original content: "For audio-only content, such as a podcast, provide a transcript."

Translation: "Untuk konten audio, seperti podcast, berikan transkripsinya"

@frirasyidi Could you specify "audio-only" (not only "audio") in your translation?

Reply:
@remibetin can and done in the next commit.